laurenlucero/VRAD
# Feature/sign in component

#### What's this PR do?
This set ups the directory with component files. It also gets a start on the sign in form as well as a basic header.
#### Where should the reviewer start?
App.js!
#### How should this be manually tested?
There is not much to test here. This PR is mostly getting the directory and files set up. It also displays a super basic header and sign in form. The sign in form can be tested by entering a user name and clicking sign in. It should display the test in the header.
#### Any background context you want to provide?
n/a
#### What are the relevant tickets?
n/a
#### Screenshots (if appropriate)
n/a
#### Questions:
n/a
